### PR TITLE
Audio poisoning

### PR DESCRIPTION
**Pull request by swsuggs**
Baseline model, attack, config, triggers

### CODE REVIEW

File: scenario_configs/eval6/poisoning/audio_backdoor_undefended.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
+    "_description": "CIFAR10 DLBD poison image classification",
```
Comment (davidslater) [inline scenario_configs/eval6/poisoning/audio_backdoor_undefended.json:2]: Rename

File: scenario_configs/eval6/poisoning/audio_backdoor_undefended.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        "name": "speech_commands",
+        "pad_data": true
+    },
+    "defense": null,
+    "metric": null,
+    "model": {
+        "fit": true,
+        "fit_kwargs": {},
+        "model_kwargs": {},
+        "module": "armory.baseline_models.tf_graph.audio_cnn",
```
Comment (davidslater) [inline scenario_configs/eval6/poisoning/audio_backdoor_undefended.json:43]: `audio_resnet50`, not `audio_cnn`?

**Comment by swsuggs** [comment]
Last thing to do is pick which triggers to include in the armory release.  First I need to complete a run with each trigger to see whether any work better than others.

**Comment by swsuggs** [comment]
Should be good to go @davidslater

**Review by davidslater — APPROVED** [review]
LGTM

**Pull request merged**